src/components/App.svelte

```diff
@@ -30,8 +30,8 @@ const authenticateUser = async () => {
     })
 }
 
-$afterPageLoad((page: { path: string }) => {
-  if (!publicRoutes.includes(page.path)) {
+$afterPageLoad((page?: ClientNodeApi) => {
+  if (!publicRoutes.includes(page?.path || 'undefined')) {
```

Comment:
What would you think of just doing a separate check first for whether we have a path value, rather than making up a path string value (`'undefined'`) that shouldn't match a real value:
```suggestion
  const path = page?.path || ''
  if (!path || !publicRoutes.includes(path)) {
```